cmd/_integration-tests/README.md

```diff
@@ -1,40 +1,39 @@
 # integration-tests
 
 These tests run truss against definition files and tests if the generated code
-behaves. 
+behaves. Each test imports and uses service code generated by `truss`, but that code doesn't exist by default and must be generated before each test runs.
 
-# ./http
+# ./transport
 
-The `http` directory contains tests for the generated HTTP client libraries and
+The `transport` directory contains tests for the generated HTTP client libraries and
 standard Go `net/http` requests.
 
 The test harness works as follows:
 
-- Runs truss against `http/httptest.proto`
-- Copy `http/handlers` into `http/handlers/httptest-service`
+- Runs truss against `transport/transport-test.proto`, which generates `transport/transport-service`
+- Copy `transport/handlers` into `transport/transport-service`
 - Run `go test -v`
-- Runs truss again against `http/httptest.proto` (for regeneration tests)
+- Runs truss again against `transport/transport-test.proto` (for regeneration tests)
 - Run `go test -v`
 
-`http/handlers` has implemented handlers for the server. They add things
+`transport/handlers` has implemented handlers for the server. They add things
 together for the purposes of testing.
 
 `http_test.go` imports the generated code, starts up a `httptest` server with
```

Comment:
Either remove this part or update it to say there is the setup_test for setting up a truss grpc and http server and there is http_test.go for http tests and grpc_test.go for grpc tests.